v2/internal/pkg/imagebuilder/rebuild_catalog.go

```diff
@@ -112,7 +113,14 @@ RUN rm -fr /tmp/cache/* && /bin/opm serve /configs --cache-only --cache-dir=/tmp
 	if err != nil {
 		return catalogCopyRefs, err
 	}
-	buildOptions, err := getStandardBuildOptions(updatedDest, srcSysCtx)
+
+	file, err := createErrorLog(filteredDir)
```

Comment:
Hi @aguidirh 
I was wondering about the location of the error file: within the filtered catalog folder? or under the working-dir/logs?

Reply:
Currently it is under the filtered catalog folder because since it is possible to include a list of catalogs to be rebuild, which one should have their own error log in order to avoid confusion.